client/src/components/PublicData/PublicData.jsx

```diff
@@ -119,6 +119,7 @@ const PublicDataListing = ({ canDownload, downloadCallback }) => {
           disabled={!canDownload}
         />
       }
+      contentLayoutName="multiColumn"
```

Comment:
Cool. I am glad the solution is so simple.

Please use `oneColumn`, because:
- it also fixes the issue ([screenshot](https://github.com/user-attachments/assets/818a3d36-759b-4950-951e-f73965a8460d))
- is a more accurate layout[^1]

[^1]: One column _of content_, where an entire table would be considered _content_. For reference: `Dashboard` uses `twoColumn` and `ManageAccount` uses `multiColumn`.